internal/service/teams/resource_teams_test.go

```diff
@@ -13,12 +13,12 @@ import (
 	"github.com/hashicorp/terraform-plugin-testing/terraform"
 	"github.com/mongodb/terraform-provider-mongodbatlas/internal/common/conversion"
 	"github.com/mongodb/terraform-provider-mongodbatlas/internal/testutil/acc"
-	matlas "go.mongodb.org/atlas/mongodbatlas"
+	"go.mongodb.org/atlas-sdk/v20231115005/admin"
 )
 
 func TestAccConfigRSTeam_basic(t *testing.T) {
 	var (
-		team         matlas.Team
+		team         admin.Team
```

Comment:
MONGODB_ATLAS_USERNAME variable is not checked in  acc.PreCheckBasic, I reccomend to create a new precheck function cheking that variable as well

Reply:
similarly for mig precheck

Reply:
good point! Done